extensions/oidc/deployment/src/main/java/io/quarkus/oidc/deployment/devservices/keycloak/KeycloakDevServicesProcessor.java

```diff
@@ -215,7 +215,7 @@ private DevServicesConfigBuildItem prepareConfiguration(boolean createRealm,
     private StartResult startContainer() {
         if (!capturedDevServicesConfiguration.enabled) {
             // explicitly disabled
-            LOG.error("Not starting devservices for Keycloak as it has been disabled in the config");
+            LOG.debug("Not starting devservices for Keycloak as it has been disabled in the config");
```

Comment:
It was `debug` in my original copy of a similar code but I changed it to `error` while I was looking at some PR problem and forgot to change it back